geode-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch

```diff
@@ -1,808 +0,0 @@
-From 15b752b4d129d4f303d033f0e6e63871355388cc Mon Sep 17 00:00:00 2001
```

Comment:
This file needs to be added back.  If you look at the output from the `execute_tests` task of the RedisTests in the pipeline, you'll see:
```
grep: geode/geode-redis/build.gradle: No such file or directory

```